Error downloading binary on alpine linux

Running into an issue when fetching the query engine on `alpine:3.11.6` docker:

```
api_1       | info: 2020/06/09 22:12:13 no query engine defined or found
api_1       | info: 2020/06/09 22:12:13 if you want to pre-fetch the query engine for better startup performance, specify `binaryTargets = ["native"]` in your Schema.prisma file under "generator" and upload the query engine with your application.
api_1       | info: 2020/06/09 22:12:13 fetching the query engine now...
api_1       | info: 2020/06/09 22:12:14 done.
api_1       | panic: ensure: version check failed: fork/exec /tmp/prisma/binaries/engines/4dbefe724ae6df3621eb660a5d0e0402e34189bf/prisma-query-engine-debian-openssl-1.1.x: no such file or directory
```

If I attach into the container I see the file is there:

```
info: 2020/06/09 23:26:17 no query engine defined or found
info: 2020/06/09 23:26:17 if you want to pre-fetch the query engine for better startup performance, specify `binaryTargets = ["native"]` in your Schema.prisma file under "generator" and upload the query engine with your application.
info: 2020/06/09 23:26:17 fetching the query engine now...
info: 2020/06/09 23:26:18 done.
panic: ensure: version check failed: fork/exec /tmp/prisma/binaries/engines/4dbefe724ae6df3621eb660a5d0e0402e34189bf/prisma-query-engine-debian-openssl-1.1.x: no such file or directory

goroutine 1 [running]:
github.com/insideiq/api/pkg/db.Init()
	/go/src/github.com/tarrencev/api/pkg/db/db.go:13 +0x1d6
main.init.0()
	/go/src/github.com/tarrencev/api/main.go:20 +0x3e
/ # ls /tmp/prisma/binaries/engines/4dbefe724ae6df3621eb660a5d0e0402e34189bf/
prisma-query-engine-debian-openssl-1.1.x      prisma-query-engine-debian-openssl-1.1.x.tmp
```

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,5 +1,6 @@
 .git/
 .github/
+.idea/
 prisma-client-go
 *.db
 *.sqlite
@@ -8,6 +9,5 @@ prisma-client-go
 schema.graphql
 **migrations**/
 **/prisma-*
-**/query-engine-*
 build/
 .graphqlconfig

--- a/binaries/platform/platform.go
+++ b/binaries/platform/platform.go
@@ -25,6 +25,11 @@ func BinaryPlatformName() string {
 	}
 
 	distro := getLinuxDistro()
+
+	if distro == "alpine" {
+		return "linux-musl"
+	}
+
 	ssl := getOpenSSL()
 
 	name := fmt.Sprintf("%s-openssl-%s", distro, ssl)
@@ -59,11 +64,11 @@ func checkForExtension(platform string, path string) string {
 func getLinuxDistro() string {
 	out, _ := exec.Command("cat", "/etc/os-release").CombinedOutput()
 
-	if out == nil {
-		return "debian"
+	if out != nil {
+		return parseLinuxDistro(string(out))
 	}
 
-	return parseLinuxDistro(string(out))
+	return "debian"
 }
 
 func parseLinuxDistro(str string) string {
@@ -80,6 +85,10 @@ func parseLinuxDistro(str string) string {
 	idLikeMatches := regexp.MustCompile(`(?m)^ID_LIKE="?([^"\n]*)"?`).FindStringSubmatch(str)
 	if len(idLikeMatches) > 0 {
 		idLike = idLikeMatches[1]
+	}
+
+	if id == "alpine" {
+		return "alpine"
 	}
 
 	if strings.Contains(idLike, "centos") ||

--- a/docker/integration.dockerfile
+++ b/docker/integration.dockerfile
@@ -24,7 +24,6 @@ RUN cd test/integration/; go build -o /app/main .
 
 # start a new stage to test if the runtime fetching works
 FROM golang:1.13
-# TODO try scratch image. golang is used because it's available on both linux and windows
 
 WORKDIR /app
 

